driver-scala/src/main/scala/org/mongodb/scala/MongoCredential.scala

```diff
@@ -64,7 +64,7 @@ object MongoCredential {
    *
    * @since 3.8
    * @note Requires MongoDB 4.0 or greater
-    @see [[http://docs.mongodb.org/manual/core/authentication/#authentication-scram-sha-256 SCRAM-SHA-256]]
+    @see [[https://www.mongodb.com/docs/manual/core/authentication/#authentication-scram-sha-256 SCRAM-SHA-256]]
```

Comment:
Looks like this links to an old target.
New link suggestion: https://www.mongodb.com/docs/manual/core/security-scram/#std-label-authentication-scram-sha-256